README.md

```diff
@@ -1,2 +1,8 @@
-# renovate
-Configuration to run Renovate as a GitHub action.
+# Renovate
+
+This repository contains the configuration to run [Renovate](https://docs.renovatebot.com/) as a GitHub action.
```

Comment:
The point is to separate running Renovate from the repositories that get updated. Copying the whole configuration to the guidance repo would have some downsides:
* If changes to the global configuration are required they have to be applied in two places.
* It would waste resources by running two separate actions on a cron schedule.
* Copying does not scale when we add more repositories later.

Of course we could also keep the action in the ort-server repo and configure it to run for other repositories as well, but I find moving it to a dedicated repository cleaner as it makes it clearer that this is a service for other repositories.

Reply:
I also overlook this part:

> Configure it to only run for this and the guidance repository **until it is removed from the ort-server repository**.